# HG changeset patch
# User Florian Pose
# Date 1538727851 -7200
#      Fri Oct 05 10:24:11 2018 +0200
Set --enable-sii-assign by default.
(grafted from b4a109b9e2a8308a46fd8151b8f85ab119620786)

diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -1057,7 +1057,7 @@ AC_MSG_CHECKING([whether to assign the S
 
 AC_ARG_ENABLE([sii-assign],
     AS_HELP_STRING([--enable-sii-assign],
-                   [Enable SII assignment to PDI (default: no)]),
+                   [Enable SII assignment to PDI (default: yes)]),
     [
         case "${enableval}" in
             yes) siiassign=1
@@ -1068,7 +1068,7 @@ AC_ARG_ENABLE([sii-assign],
                 ;;
         esac
     ],
-    [siiassign=0]
+    [siiassign=1]
 )
 
 if test "x${siiassign}" = "x1"; then
